website/addons/mendeley/static/node-cfg.js

```diff
@@ -1,7 +1,6 @@
 'use strict';
 
 var CitationsNodeConfig = require('js/citationsNodeConfig').CitationsNodeConfig;
-require('./node-cfg.css');
```

Comment:
@sloria node-cfg.css is a empty css file, which is not used at anywhere. So I think it is OK to delete them.